packages/authentication/docs/controller-functions.md

```diff
@@ -5,7 +5,7 @@ the beginning of markdown file
 [authentication-system](./authentication-system.md).
 
 Please note how they are decorated with `@authenticate()`, the syntax is:
-`@authenticate(<strategy_name>, {action: <action_name>, session: <enabled_or_not>})`
+`@authenticate(strategy_name, options)`
```

Comment:
In the future, we can allow class-level decoration of `@authenticate`.

Reply:
Agree. sounds like an improvement could be applied with https://github.com/strongloop/loopback-next/pull/2576#discussion_r269311285